clients/include/rocblas_matrix.hpp

```diff
@@ -221,6 +221,13 @@ inline void rocblas_init_matrix(host_matrix<T>&           hA,
         else
             rocblas_init_matrix(matrix_type, arg.uplo, random_generator<T>, hA);
     }
+    else if(arg.initialization == rocblas_initialization::rand_int_zero_one)
+    {
+        if(alternating_sign)
+            rocblas_init_matrix_alternating_sign(matrix_type, arg.uplo, random_generator<T>, hA);
```

Comment:
Probably not used but I think this should have `random_zero_one_generator<T>` instead of `random_generator<T>`, although maybe the two are just incompatible.